monitoring/README.md

```diff
@@ -86,7 +90,7 @@ pyroscope-alloy-0                                      2/2     Running   0
 a) WDS space:
 
 ```bash
-./configure-monitoring-wds.sh
+./configure-monitoring-wds.sh --kubeflex-hosting-cluster-context kind-kubeflex
 ```
 
 If you have more than one WDS, you can run the above script with the `--space-name` flag. For example: `./configure-monitoring-wds.sh --space-name wds2`
```

Comment:
I think this PR did not introduce this problem, but I encountered it while testing. The problem is the assumption that a space from a KubeFlex ControlPlane named "foo" is accessed through a kubeconfig context named "foo". That is a convenient pattern that we generally follow, but is not a requirement elsewhere in KubeStellar. Now that I am working with three different systems (two triples of clusters in the IBM cloud, plus local testing with a triple of `kind` clusters), I can not use the one context name "wds1" for three different contexts. I am currently testing with a system where I use a context named "dev-wds1" to access the space defined by the ControlPlane named "wds1". Following is a failure.

```console
bash-3.2$ time ./configure-monitoring-wds.sh --kubeflex-hosting-cluster-context $host_context --space-name dev-wds1
+ set -e
+ ctx=kind-kubeflex
+ wds=wds1
+ monitoring_ns=ks-monitoring
+ '[' 4 '!=' 0 ']'
+ case "$1" in
+ ((  4 > 1  ))
+ ctx=default/c106-e-us-south-containers-cloud-ibm-com:31891/IAM#mspreitz@us.ibm.com
+ shift
+ shift
+ '[' 2 '!=' 0 ']'
+ case "$1" in
+ ((  2 > 1  ))
+ wds=dev-wds1
+ shift
+ shift
+ '[' 0 '!=' 0 ']'
+++ dirname -- ./configure-monitoring-wds.sh
++ cd -- .
++ pwd
+ SCRIPT_DIR=/Users/mspreitz/go/src/github.com/kubestellar/kubestellar/monitoring
+ kubectl config use-context default/c106-e-us-south-containers-cloud-ibm-com:31891/IAM#mspreitz@us.ibm.com
Switched to context "default/c106-e-us-south-containers-cloud-ibm-com:31891/IAM#mspreitz@us.ibm.com".
+ : --------------------------------------------------------------------
+ : Configure APF for prometheus traffic to the WDS API server
+ : --------------------------------------------------------------------
+ kubectl --context dev-wds1 apply -f /Users/mspreitz/go/src/github.com/kubestellar/kubestellar/monitoring/configuration/prometheus-pod-exempt.yaml
Warning: flowcontrol.apiserver.k8s.io/v1beta3 FlowSchema is deprecated in v1.29+, unavailable in v1.32+; use flowcontrol.apiserver.k8s.io/v1 FlowSchema
flowschema.flowcontrol.apiserver.k8s.io/prometheus-exempt created
+ : --------------------------------------------------------------------
+ : Configure kubestellar controller manager pod for prometheus scraping
+ : --------------------------------------------------------------------
+ : 1. Create service monitor for KS ctl manager
+ sed s/%WDS%/dev-wds1/g /Users/mspreitz/go/src/github.com/kubestellar/kubestellar/monitoring/configuration/ks-ctl-manager-sm.yaml
+ kubectl -n ks-monitoring apply -f -
servicemonitor.monitoring.coreos.com/dev-wds1-ks-controller-manager created
+ : --------------------------------------------------------------------
+ : Configure kubestellar controller manager pod for pyroscope scraping
+ : --------------------------------------------------------------------
+ : 1. Add labels for pyroscope scraping:
+ kubectl -n dev-wds1-system get deploy kubestellar-controller-manager -o yaml
+ yq '.spec.template.metadata.annotations."profiles.grafana.com/cpu.port" |= "8082" |  .spec.template.metadata.annotations."profiles.grafana.com/cpu.scrape"|= "true" | .spec.template.metadata.annotations."profiles.grafana.com/goroutine.port" |= "8082" | .spec.template.metadata.annotations."profiles.grafana.com/goroutine.scrape" |= "true" |
.spec.template.metadata.annotations."profiles.grafana.com/memory.port" |= "8082" | .spec.template.metadata.annotations."profiles.grafana.com/memory.scrape" |= "true"'
+ kubectl -n dev-wds1-system apply -f -
Error from server (NotFound): namespaces "dev-wds1-system" not found
error: error validating "STDIN": error validating data: [apiVersion not set, kind not set]; if you choose to ignore these errors, turn validation off with --validate=false

real	0m2.812s
user	0m0.391s
sys	0m0.207s
```